lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -69,105 +92,25 @@ def _get_results_s3_key(self, job_id: UUID) -> str:
             # if no config template is provided, get the default one for the model
-            config_template = config_templates.config_infer_template[request.model]
+            config_template = job_templates[model_name]
         else:
             # if no default config template is provided, get the causal template
             # (which works with seq2seq models too except it does not use pipeline)
-            config_template = config_templates.causal_infer_template
-
-        infer_config_args = {
-            "--config": config_template.format(**config_params),
-        }
-
-        # Prepare the job configuration that will be sent to submit the ray job.
-        # This includes both the command that is going to be executed and its
-        # arguments defined in infer_config_args
-        ray_config = JobConfig(
-            job_id=record.id,
-            job_type=JobType.INFERENCE,
-            command=settings.INFERENCE_COMMAND,
-            args=infer_config_args,
-        )
+            config_template = job_templates["default"]
 
-        # build runtime ENV for workers
-        runtime_env_vars = {"MZAI_JOB_ID": str(record.id)}
-        settings.inherit_ray_env(runtime_env_vars)
-
-        # set num_gpus per worker (zero if we are just hitting a service)
-        if not request.model.startswith("hf://"):
-            worker_gpus = settings.RAY_WORKER_GPUS_FRACTION
-        else:
-            worker_gpus = settings.RAY_WORKER_GPUS
-
-        runtime_env = {
-            "pip": settings.INFERENCE_PIP_REQS,
-            "working_dir": settings.INFERENCE_WORK_DIR,
-            "env_vars": runtime_env_vars,
-        }
-
-        loguru.logger.info("runtime env setup...")
-        loguru.logger.info(f"{runtime_env}")
-
-        entrypoint = RayJobEntrypoint(
-            config=ray_config, runtime_env=runtime_env, num_gpus=worker_gpus
-        )
-        loguru.logger.info("Submitting Ray job...")
-        submit_ray_job(self.ray_client, entrypoint)
-
-        loguru.logger.info("Getting response...")
-        return JobResponse.model_validate(record)
-
-    def create_evaluation_job(self, request: JobCreate) -> JobResponse:
-        """Creates a new evaluation workload to run on Ray and returns the response status"""
-        # Create a db record for the job
-        record = self.job_repo.create(name=request.name, description=request.description)
+        return config_template
 
+    def _get_job_params(self, job_type: str, record, request: BaseModel) -> dict:
         # get dataset S3 path from UUID
         dataset_s3_path = self.data_service.get_dataset_s3_path(request.dataset)
 
-        # set storage path
-        storage_path = f"s3://{ Path(settings.S3_BUCKET) / settings.S3_JOB_RESULTS_PREFIX }/"
-
-        # fill up model url with default openai url
+        # TODO: the following is currently both eval and inference, but
+        #       will soon be inference only. Make sure we'll move this out
```

Comment:
❓ same question for this `TODO` 😄 could we create an issue to track the required work if we need to keep note of this?

Reply:
No need, this is gonna be part of my next PR after Vicki's code is merged too